mysql-test/suite/tokudb_rpl/r/rpl_tokudb_row_crash_safe.result

```diff
@@ -195,6 +195,8 @@ END|
 
 
 STOP SLAVE SQL_THREAD;
+Warnings:
```

Comment:
Here and everywhere in this file
Probably it's better to change the command in the .test file

Reply:
Explained above.